tasks/github_tasks.py

```diff
@@ -421,3 +422,33 @@ def assign_codereview_label(_, pr_id=-1):
     gh = GithubAPI('DataDog/datadog-agent')
     complexity = gh.get_codereview_complexity(pr_id)
     gh.update_review_complexity_labels(pr_id, complexity)
+
+
+@task
+def agenttelemetry_list_change_ack_check(_, pr_id=-1):
+    """
+    Change to `comp/core/agenttelemetry/impl/config.go` file requires to acknowledge
+    potential changes to Agent Telemetry metrics. If Agent Telemetry metric list has been changed,
+    the PR should be labeled with `need-change/agenttelemetry-governance` and follow
+    `Agent Telemetry Governance` instructions to potentially perform additional changes. See
+    https://datadoghq.atlassian.net/wiki/spaces/ASUP/pages/4340679635/Agent+Telemetry+Governance
+    for details.
+    """
+    from tasks.libs.ciproviders.github_api import GithubAPI
+
+    gh = GithubAPI('DataDog/datadog-agent')
+
+    pr_id = int(os.environ.get("PR_ID"))
+
+    labels = gh.get_pr_labels(pr_id)
+    files = gh.get_pr_files(pr_id)
+    if "comp/core/agenttelemetry/impl/config.go" in files:
+        if "need-change/agenttelemetry-governance" not in labels:
+            message = f"{color_message('Error', 'red')}: If you change the `comp/core/agenttelemetry/impl/config.go` file, you need to add `need-change/agenttelemetry-governance` label. Please follow the instructions specified in https://datadoghq.atlassian.net/wiki/spaces/ASUP/pages/4340679635/Agent+Telemetry+Governance"
```

Comment:
As we discussed this document won't be accessible for external contributors. So I don't know it this suggestion makes sense
```suggestion
            message = f"{color_message('Error', 'red')}: If you change the `comp/core/agenttelemetry/impl/config.go` file, you need to add `need-change/agenttelemetry-governance` label. If you have access, please follow the instructions specified in https://datadoghq.atlassian.net/wiki/spaces/ASUP/pages/4340679635/Agent+Telemetry+Governance"
```
It might need an `otherwise` clause for external contributor (contact us with an issue?)
This suggestion can be discarded if it's irrelevant

Reply:
This concerns me most of all. However ...
- I do not think we want, at least in this repo, external contributors to touch the file without very very good reason. 
- I have searched for `https://datadoghq.atlassian.net/wiki` and found a handful of usage in other places
- More importantly how I can tell PR submitter where to look without telling were to look? Do you have any suggestions?

Reply:
No I don't have a great suggestion here. As you say, if external contributions are not expected I think we are fine with an internal link.